packages/components/src/components/modal/modal.scss

```diff
@@ -1,5 +1,8 @@
 /** @define dc-modal; weak */
 .dc-modal {
+    #dt_set_account_currency_modal .dc-btn--secondary {
```

Comment:
don't think this belongs in `components` as they shouldn't handle package specific of implementation details

Reply:
Yes. Moved to its own style file.